app/sportEvaluator.py

```diff
@@ -61,9 +61,11 @@
 
 # region Weights
 
+1 -0.1 -0.1 -0.5 -0.8
```

Comment:
Me I'm not understanding this line

Reply:
It's used for quantum computing the weights